### PR TITLE
Make update commands alert to sentry on exceptions

### LINKED ISSUE #10813: DB update script - improve logging
#### Success Criteria
- [ ] Each step of the DB update script should log any failures to Sentry

### PR DESCRIPTION
**Pull request by stevejalim**
## Description

This is part of [a broader epic](https://github.com/mozilla/bedrock/issues/10627) about making the DB update script a bit more resilient and its results more visible

This changeset adds a way for us to have management commmands raise exceptions upon failure, but still capture that info in Sentry, in a fairly unobstrusive way, via a class decorator

## Issue / Bugzilla link

Resolves #10813 

## Testing

* Add the SENTRY_DSN from Dev to your .env and restart your local stack
* In one of the relevant management commands (eg `update_contentful`) raise an Exception high in the _handle() command
* Confirm the exception is shown in your console AND also gets sent to sentry - eg https://sentry.prod.mozaws.net/operations/bedrock-dev/issues/15307834/?referrer=alert_email

### CODE REVIEW

File: bedrock/utils/management/command_base.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        """Custom wrapper for management commands that failures
+        anywhere in the main handle() call get to Sentry.
+
+        self._handle is an expected method of any subclass of SentryAlertingBaseCommand.
+
+        Usage: in your management command, rename/create the handle() method so that
+        it is _handle()
+        """
+        try:
+            self._handle(*args, **options)
```
Comment (pmac) [inline bedrock/utils/management/command_base.py:21]: Another option, if you don't want to have to change the name of all of the `handle()` methods, is to override the `execute()` method instead, which is the method in the `BaseCommand` class that calls `self.handle()`. Not required, just another idea.

**Comment by stevejalim** [comment]
@pmac Refactored and am a lot happier with this approach. Sending up the batsignal for re-review :bat:

File: bedrock/utils/management/cron_command.py [hunk 2]
```diff
@@ -14,6 +14,8 @@
 from django.core.cache import cache
 from django.core.management.base import BaseCommand
 
+from bedrock.utils.management.decorators import alert_sentry_on_exception
```
Comment (pmac) [inline bedrock/utils/management/cron_command.py:17]: I'm not even sure we need this `CronCommand` class any longer. I think this is from a time before k8s and SQLite. Only the security advisories command is using it.

Comment (pmac) [reply bedrock/utils/management/cron_command.py:17]: Probably out of scope for this PR, but I don't think this class needs the decorator, only the `update_security_advisories.py` one does.

Comment (stevejalim) [reply bedrock/utils/management/cron_command.py:17]: Will ticket it for the grab bag

**Review by robhudson — APPROVED** [review]
Seems like a nice solution.

**Review by pmac — APPROVED** [review]
🏖️

**Pull request merged**